azext_edge/edge/providers/check/common.py

```diff
@@ -327,4 +327,16 @@ class DataLakeConnectorTargetType(ListableEnum):
 
 MIN_K8S_VERSION = "1.20"
 
+# Node prerequisite constants
+
+MIN_NODE_MEMORY = "16G"
```

Comment:
out of curiosity, where did we find these requirements value?😋

Reply:
The wonderful work item (where we needed to make sure we do not need a machine to run COD)